content/2022/chinese/_index.md

```diff
@@ -47,9 +47,19 @@ With the participation of Asia Info,IBM,TongTech,Apusic,Cvcise and other develop
   {{</ grid/div >}}
 {{</ grid/section-container >}}
 
+<<<<<<< HEAD
```

Comment:
merge error

Reply:
yeah, saw that this morning, rebasing since @laliwa pushed another patch this morning.